tools/replace-next-version-tag.sh

```
@@ -0,0 +1,114 @@
+
+function debug {
+	[[ -n "$VERBOSE" ]] && echo "$@" >&2
+	return 0
+}
+
+# Run from the repo root so the path argument resolves predictably.
+cd "$(git rev-parse --show-toplevel)"
+
+EXIT=0
+for FILE in $(git ls-files "$PATH_ARG"); do
+	grep -F -q '$$next-version$$' "$FILE" 2>/dev/null || continue
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)